Marlin/src/lcd/extui/lib/mks_ui/draw_extrusion.cpp

```diff
@@ -195,7 +195,7 @@ void disp_ext_speed() {
 
 void disp_hotend_temp() {
   char buf[20] = {0};
-  sprintf(buf, extrude_menu.temp_value, (int)thermalManager.degHotend(uiCfg.extruderIndex), (int)thermalManager.degTargetHotend(uiCfg.extruderIndex));
+  sprintf(buf, extrude_menu.temp_value, thermalManager.degHotend(uiCfg.extruderIndex), thermalManager.degTargetHotend(uiCfg.extruderIndex));
```

Comment:
you need cast to int here if the code uses `%d`... in fact, is safe to keep the cast! I think we need/should keep the cast to int.